dev/create_keycloak_db.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
License and Copyright header missing, please refer to [TRG 7.02 - License and Copyright header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02).

Reply:
fixed with https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/72/commits/5dae4a27379df156672d326210033489a710b240

Reply:
Hm, `#!/bin/bash` should be in in Line 1 to work properly. Maybe you should also consider to use `#!/usr/bin/env bash` to cover certain install location of Bash.